docs/userguides/legalhold.md

```diff
@@ -0,0 +1,96 @@
+# Manage legal hold custodians
+
+Once you [create a legal hold matter in the Code42 console](https://support.code42.com/Administrator/Cloud/Configuring/Create_a_legal_hold_matter#Step_1:_Create_a_matter), you can use the Code42 CLI to add or release custodians from the matter.
+
+Use the `legal-hold` commands to manage legal hold custodians.
+ - To see a list of all the users currently in your organization, you can export a list from the [Users action menu](https://support.code42.com/Administrator/Cloud/Code42_console_reference/Users_reference#Action_menu).
+ - To view a list of legal hold matters for your organization, including the matter ID, use the following command:
+   `code42 legal-hold list`
+ - To see a list of all the custodians currently associated with a legal hold matter, enter `code42 legal-hold show <matterID>`.
+
+
+## Get CSV template
+
+To add multiple custodians to a legal hold matter:
+
+1. Generate a CSV template. Below is an example command for generating a template to use to add custodians to legal hold matter. Once generated, the CSV file is saved to your current working directory.
+    `code42 legal-hold bulk generate-template add`
+
+    To generate a template to use to remove custodians from a legal hold matter, for example:
+
+    `code42 legal-hold bulk generate-template remove`
+
+    The CSV templates for adding and removing are the same.
+
+2. Use the CSV template to enter the matter ID(s) and Code42 usernames for the custodians you want to add to the matters.
+To get the ID for a matter, enter `code42 legal-hold list`.
+3. Save the CSV file.
+
+## Add custodians to a legal hold matter
+
+You can add one or more custodians to a legal hold matter using the Code42 CLI.
+
+### Add multiple custodians
+Once you have entered the matter ID and user information in the CSV file, use the bulk add-user command with the CSV file path to add multiple custodians at once. For example:
+
+`code42 legal-hold bulk add-user /Users/astrid.ludwig/add_users_to_legal_hold.csv`
+
+### Add a single custodian
+
+To add a single custodian to a legal hold matter, use the following command as an example:
+
+`code42 legal-hold add-user --matter-id 123456789123456789 --username tanya.rowe@example.com`
```

Comment:
unless Tanya Rowe has no objections of her name being used. ;)

Reply:
this is not a real person's email address (although a person with that name probably does exist), these are names that are generated and appear in our demo environment.